will_kitchell/server.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var fs = require('fs');
+var express = require('express');
+var app = express();
+
+app.set('port', (process.env.PORT || 5000));
+
+var handleData = function(req, res, next) {
+  var data = '';
+
+  req.on('data', function(reqData) {
+    data = data + reqData.toString();
+  });
+
+  req.on('end', function(endData) {
+    req.body = data;
+    next();
+  });
+};
```

Comment:
It's a small project, but ideally your handleData function would be modularized.